packages/mobile/src/index.tsx

```diff
@@ -8,10 +8,12 @@ import { store } from '~/store/store';
 import { NavigationDarkTheme, NavigationTheme } from '~/config/config';
 import { useColorScheme } from '~/hooks/hooks';
 import { i18 } from './localization/localization';
+import { dayjsInit } from './day-js/day-js-init';
```

Comment:
We don't need that root folder. Use helpers or create some sort of AppService with init() method instead

Reply:
done